From 453cd904025640580582a278690de5ba3d38119d Mon Sep 17 00:00:00 2001
From: Allen Xu <b45815@freescale.com>
Date: Thu, 25 Sep 2014 16:08:16 -0500
Subject: [PATCH] MLK-9650 QSPI: imx6sx: fix QSPI build warning

add the missed return value to fix the build warning.

Signed-off-by: Allen Xu <b45815@freescale.com>
(cherry picked from commit a4e7c495626ec3a0cbe9561fa5a22306a92ca2bf)
---
 drivers/mtd/spi-nor/fsl-quadspi.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/mtd/spi-nor/fsl-quadspi.c b/drivers/mtd/spi-nor/fsl-quadspi.c
index fafc8ad..27b9c4e 100644
--- a/drivers/mtd/spi-nor/fsl-quadspi.c
+++ b/drivers/mtd/spi-nor/fsl-quadspi.c
@@ -672,7 +672,7 @@ static int fsl_qspi_clk_disable_unprep(struct fsl_qspi *q)
 {
 	clk_disable_unprepare(q->clk);
 	clk_disable_unprepare(q->clk_en);
-
+	return 0;
 }
 
 /* We use this function to do some basic init for spi_nor_scan(). */
-- 
1.8.0

